legion/engine/physics/systems/physics_test_system.cpp

```diff
@@ -0,0 +1,910 @@
+#include <physics/systems/physics_test_system.hpp>
+#include <physics/mesh_splitter_utils/mesh_splitter.hpp>
+#include <physics/mesh_splitter_utils/splittable_polygon.hpp>
+#include <physics/components/physics_component.hpp>
+#include <physics/systems/physicssystem.hpp>
+#include <physics/components/fracturer.hpp>
+#include <rendering/debugrendering.hpp>
+#include <rendering/components/camera.hpp>
+#include <rendering/rendering.hpp>
+#include <physics/components/fracturecountdown.hpp>
+
+namespace legion::physics
+{
+    void PhysicsTestSystem::setup()
```

Comment:
this system uses the gfx module pretty intensely. It's not a problem for now but something like this should normally end up in an application rather than inside of the module's source

Reply:
This will be fixed in #389 where it is moved to Sandbox. I prefer to do it there since this current PR is already relatively large.